# Fix: Update CHANGELOG.md date from 2024 to 2025

This patch corrects the release date in CHANGELOG.md.

--- CHANGELOG.md
+++ CHANGELOG.md
@@ -5,7 +5,7 @@ All notable changes to the Git ADR extension will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [0.1.0] - 2024-12-17
+## [0.1.0] - 2025-12-17
 
 ### Added
 - Initial release of Git ADR for VS Code
